client/src/featureform/local_test.py

```diff
@@ -199,3 +195,25 @@ def transform3(df):
     features=[("SepalLength", "join"), ("SepalWidth", "join"), ("PetalLength", "join"),
               ("PetalWidth", "join")],
 )
+
+def add_invalid_label():
```

Comment:
If you want to use this as a pytest file, you should remove the other register functions from above. Seems like a weird hybrid definitions + test file instead of one or the other right now. Or just make everything a test function